### ISSUE TITLE
Transcoding problems with Java version >1.8.0_60

### ISSUE THREAD

**Issue by Nadahar** [UniversalMediaServer/UniversalMediaServer#751]
@UniversalMediaServer/developers @SubJunk 
This has come up multiple times in the support forum lately, and we've discussed it on GitHub as well. I still feel this needs more attention, as a lot of people use Java's auto update feature. I suspect it will lead to both lots of support cases and abandonment of UMS by many.

What seems to be the  problem is the creation of pipes under Windows, used to communicate with the transcoder. I have no clue what is the problem, but I'm suspecting it's some new "security" measure they've implemented.

**Comment by SubJunk** [comment]
Hmm, I use 1.8.0_66 on Windows 7 and haven't noticed any issues with transcoding, so it may be a hard one of reproduce. Have you noticed whether users are using a specific version of Windows, or whether it's 32 or 64-bit Java?

**Comment by Nadahar** [comment]
@valib said he had experienced the bug, but that it went away "by itself". Most of the cases I've seen has been Windows 10, but I've also seen it reported on Windows 7. The reports are on the forum, but the limited (forum) search possibilities make them hard to find.

From what I've seen, what characterizes this problem is the following log entry:
```logtalk
java.lang.IllegalArgumentException: Unrecognized calling convention: 3
```

The first case I saw was [this one] (http://www.universalmediaserver.com/forum/viewtopic.php?t=5559) with ```1.8.0_65``` but I've later seen the same problem reported with ```1.8.0_66``.

@valib's report that it went away makes me wonder if the problem is related to what JDK version that was used to build. If that's the case, we who build ourselves before running won't notice it, but users downloading prebuilt versions will. I've read the changelog for the JRE/JDK, but can't fine anything remotely related. But, everything is probably not in the changelog.

@SubJunk: completely unrelated, I really need that you look at my forum thread/PM.

**Comment by valib** [comment]
@Nadahar I use the Java 1.8.0_66 JRE and JDK on Windows 10 and I like @SubJunk also newer seen such problem.

**Comment by Nadahar** [comment]
@valib Then I have misunderstood something. I though you said earlier that you experienced the problem after upgrading to 1.8.0_66, but that is went away. I can't remember where though...

I still think it's strange when people can downgrade to 1.8.0_60 and it starts working, while re-upgrading to 1.8.0_66 makes it stop working again. But you may be right, I just thought I'd seen it quite a lot lately, but I'm having trouble finding the actual cases.

**Comment by Nadahar** [comment]
Actually, when I look at it this issue is more or less a duplicate of #714, but the issue remains unsolved so I'll leave it open representing independent reports of the same on the forum.

**Comment by SubJunk** [comment]
@Nadahar do you mean the threadsafe one? I'm ready to add the logo to our About page if you want

**Comment by Nadahar** [comment]
@SubJunk Yes, if you add the logo I'll send in the agreement, and we should be good to go.

**Comment by SubJunk** [comment]
@Nadahar it's up

**Comment by valib** [comment]
@Nadahar ahh you are right but because it occures only once for me and immediatelly disappeared I have forgoten it. As I wrote before I don't have any problem since that time.

**Comment by Nadahar** [comment]
@valib No, but for others it stays this way...

**Comment by Sami32** [comment]
@SubJunk Did you fix https://github.com/UniversalMediaServer/UniversalMediaServer/issues/714 fixed this issue as well ?

**Comment by SubJunk** [comment]
@Sami32 I'm not sure but we haven't had this type of report lately on the forum so maybe. It can be re-opened if there's activity

**Comment by Sami32** [comment]
@SubJunk Look reasonnable to me.

**Comment by Nadahar** [comment]
This ended up being an issue with the then latest version of JNA and 64 bit OS'es. It was solved by downgrading JNA until they patched the bug. The connection with the Java version was wrong, timing og events simply made it seem likt that was a possible culprit.

My bad for forgetting to close this.

**Comment by Sami32** [comment]
@Nadahar Don't worry, it's happened to everyone here :-)
